src/BackpackServiceProvider.php

```diff
@@ -129,6 +130,25 @@ public function registerMiddlewareGroup(Router $router)
         if (config('backpack.base.setup_password_recovery_routes')) {
             $router->aliasMiddleware('backpack.throttle.password.recovery', ThrottlePasswordRecovery::class);
         }
+
+        // register the verified middleware in the backpack middleware group
+        // but only if functionality is enabled by developer in config
+        if (config('backpack.base.setup_email_verification_routes', false)) {
+            // developers that updated to 10.x but didn't add the aliases into their Kernel.php
+            // would get an error when trying to use the verified/signed middlewares
+            // we ensure they'r properly aliased if not previously set by developer
+            $routeMiddlewares = $router->getMiddleware();
+
+            if (! in_array('verified', array_keys($routeMiddlewares))) {
+                $router->aliasMiddleware('verified', \Illuminate\Auth\Middleware\EnsureEmailIsVerified::class);
+            }
+
+            if (! in_array('signed', array_keys($routeMiddlewares))) {
+                $router->aliasMiddleware('signed', \Illuminate\Routing\Middleware\ValidateSignature::class);
+            }
```

Comment:
I guess what triggers my spidey sense might be the fact that we do this here, in `BackpackServiceProvider`. So if the config is `true`, we add these aliasMiddleware to all requests, for both app and admin panel. Not just the email-verification requests. So... the scope of this fix is much bigger than the email-verification routes.

Reply:
We can add the route check to add this only in backpack routes, after all it's the only place where we will be using it. 

I have no concerns in adding this to the docs and removing it from here. It's a simple step and it will not be required everytime.